cirq/aqt/aqt_device.py

```diff
@@ -62,6 +65,15 @@ def __init__(self):
 
     def noisy_moment(self, moment: ops.Moment,
                      system_qubits: Sequence[ops.Qid]):
+        """Returns a list of noisy moments including
```

Comment:
I keep reading `Depolarizing noise with gate-dependent strength Crosstalk  between neighboring qubbits` as a sentence which is confusing me. Also there is a typo for `qubits` here.